salt/modules/daemontools.py

```diff
@@ -201,3 +212,46 @@ def get_all():
         raise CommandExecutionError("Could not find service directory.")
     #- List all daemontools services in
     return sorted(os.listdir(SERVICE_DIR))
+
+
+def enabled(name, **kwargs):
+    '''
+    Return True if the named service is enabled, false otherwise
+    A service is considered enabled if in your service directory:
+    - an executable ./run file exist
+    - a file named "down" does not exist
+
```

Comment:
@silenius, since we're backporting this, this function needs a `.. versionadded:: 2015.5.6` here.